lighthouse-cli/test/smokehouse/frontends/smokehouse-bin.js

```diff
@@ -75,6 +76,47 @@ function getDefinitionsToRun(allTestDefns, requestedIds, {invertMatch}) {
   return smokes;
 }
 
+/**
+ * Parses the cli `shardArg` flag into `shardNumber/shardTotal`. Splits
+ * `testDefns` into `shardTotal` shards and returns the `shardNumber`th shard.
+ * Shards will differ in size by at most 1.
+ * Shard params must be 1≤shardNumber≤shardTotal.
```

Comment:
here and elsewhere: can you break up with spaces, hard to read.  `1 ≤ shardNumber ≤ shardTotal`

Reply:
good call